src/assets/mapStyles/waterTemperature/mapStyles.js

```diff
@@ -45,11 +45,27 @@ export default {
             {
                 'id': 'background',
                 'paint': {
-                    'background-color': 'hsl(205, 56%, 73%)'
+                    'background-color': 'hsl(47, 26%, 88%)'
                 },
                 'type': 'background',
                 'showButtonLayerToggle': false
             },
+            {
```

Comment:
adds oceans and also inland waters, but we cover up the inland waters within the US with our states fill layer